pkg/operator/ceph/cluster/crash/crash.go

```diff
@@ -46,7 +46,7 @@ import (
 
 const (
 	crashCollectorKeyringUsername = "client.crash"
-	crashCollectorSecretName      = "rook-ceph-crash-collector-keyring"
+	crashCollectorScrtName        = "rook-ceph-crash-collector-keyring"
```

Comment:
why?

Reply:
Apparently Secret is a keyword that gets triggered by gosec. 

Variables are considered to look like a password if they have match any one of:

```- “password”
- “pass”
- “passwd”
- “pwd”
- “secret”
- “token”```

Note: this can be noisy and may generate false positives.

Reply:
@leseb